test/elm/arithmetic/arithmetic-test.ts

```diff
@@ -238,6 +239,7 @@ describe('MinValue', () => {
   });
 
   it('of Decimal should return minimum representable Decimal value', function () {
+    // eslint-disable-next-line @typescript-eslint/no-loss-of-precision
```

Comment:
Ugh. Yeah.  So this reminds me.  Some day we need to introduce a `BigDecimal` type (or something like it) so we don't lose precision (and so we can distinguish between `1` and `1.0` and `1.00`).  But not today.

Reply:
Ha, yep. Another nice improvement that we can make to the library that typescript is so kindle reminding us of :)